dependencies.md

```diff
@@ -27,66 +27,68 @@
+| [Apache Maven Install Plugin][61]                       | [Apache License, Version 2.0][1]          |
+| [Apache Maven Site Plugin][63]                          | [Apache License, Version 2.0][2]          |
 
-[15]: https://github.com/exasol/project-keeper-maven-plugin
 [27]: https://www.eclemma.org/jacoco/index.html
-[24]: http://www.apache.org/licenses/LICENSE-2.0
+[51]: https://github.com/exasol/project-keeper-maven-plugin
+[16]: http://www.apache.org/licenses/LICENSE-2.0
 [1]: http://www.apache.org/licenses/LICENSE-2.0.txt
 [21]: https://maven.apache.org/surefire/maven-surefire-plugin/
-[45]: http://www.sonatype.com/public-parent/nexus-maven-plugins/nexus-staging/nexus-staging-maven-plugin/
+[43]: http://www.sonatype.com/public-parent/nexus-maven-plugins/nexus-staging/nexus-staging-maven-plugin/
+[15]: http://www.scalatest.org
 [9]: https://github.com/mockito/mockito
-[16]: https://opensource.org/licenses/MIT
-[39]: https://maven.apache.org/surefire/maven-failsafe-plugin/
+[37]: https://maven.apache.org/surefire/maven-failsafe-plugin/
+[52]: https://opensource.org/licenses/MIT
 [29]: http://www.mojohaus.org/versions-maven-plugin/
 [14]: http://opensource.org/licenses/BSD-3-Clause
 [19]: https://maven.apache.org/plugins/maven-compiler-plugin/
-[41]: http://maven.apache.org/plugins/maven-gpg-plugin/
-[55]: https://maven.apache.org/plugins/maven-resources-plugin/
+[39]: http://maven.apache.org/plugins/maven-gpg-plugin/
+[57]: https://maven.apache.org/plugins/maven-resources-plugin/
 [35]: https://github.com/itsallcode/openfasttrace-maven-plugin
-[53]: https://maven.apache.org/plugins/maven-clean-plugin/
+[55]: https://maven.apache.org/plugins/maven-clean-plugin/
 [28]: https://www.eclipse.org/legal/epl-2.0/
 [5]: https://www.scala-lang.org/
-[43]: https://maven.apache.org/plugins/maven-deploy-plugin/
-[46]: http://www.eclipse.org/legal/epl-v10.html
+[41]: https://maven.apache.org/plugins/maven-deploy-plugin/
+[44]: http://www.eclipse.org/legal/epl-v10.html
 [18]: http://unlicense.org/
 [6]: https://www.apache.org/licenses/LICENSE-2.0
-[37]: http://zlika.github.io/reproducible-build-maven-plugin
-[61]: https://maven.apache.org/plugins/maven-site-plugin/
+[49]: http://zlika.github.io/reproducible-build-maven-plugin
+[63]: https://maven.apache.org/plugins/maven-site-plugin/
 [36]: https://www.gnu.org/licenses/gpl-3.0.html
 [0]: https://parquet.apache.org
 [2]: https://www.apache.org/licenses/LICENSE-2.0.txt
-[23]: https://github.com/scalatest/scalatest-maven-plugin
+[23]: http://nexus.sonatype.org/oss-repository-hosting.html/scalatest-maven-plugin
 [33]: https://maven.apache.org/enforcer/maven-enforcer-plugin/
 [10]: https://github.com/mockito/mockito/blob/release/3.x/LICENSE
 [8]: https://www.eclipse.org/legal/epl-v20.html
-[3]: https://github.com/google/guava
-[59]: http://maven.apache.org/plugins/maven-install-plugin/
+[3]: https://github.com/google/guava/guava
```

Comment:
I think this fails with link checker. This Guava link is somehow wrong from dependency.

Reply:
Yea, I created a PR: https://github.com/google/guava/pull/5619